scripts/enable_automatic_updates.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+apt-get install unattended-upgrades curl
+
+echo '
+APT::Periodic::Update-Package-Lists "1";
+APT::Periodic::Unattended-Upgrade "1";
+' >/etc/apt/apt.conf.d/20auto-upgrades
+
+sed -i.bak 's|//Unattended-Upgrade::Remove-Unused-Dependencies "false"|Unattended-Upgrade::Remove-Unused-Dependencies "true"|' /etc/apt/apt.conf.d/50unattended-upgrades
+
+sed -i.bak 's|//Unattended-Upgrade::Automatic-Reboot "false"|Unattended-Upgrade::Automatic-Reboot "false"|' /etc/apt/apt.conf.d/50unattended-upgrades
+
+rm /etc/apt/apt.conf.d/50unattended-upgrades.bak
+
+echo '
+APT::Periodic::Update-Package-Lists "1";
+APT::Periodic::Download-Upgradeable-Packages "1";
+APT::Periodic::AutocleanInterval "1";
+APT::Periodic::Unattended-Upgrade "1";
+APT::Periodic::Verbose 2;
+APT::Periodic::RandomSleep 1;
+'> /etc/apt/apt.conf.d/10periodic
+
+export VAULT_ADDR=https://support.montagu.dide.ic.ac.uk:8200
+if [ "$VAULT_AUTH_GITHUB_TOKEN" = "" ]; then
+    echo -n "Please provide your GitHub personal access token for the vault: "
+    read -s token
+    echo ""
+    export VAULT_AUTH_GITHUB_TOKEN=${token}
+fi
+vault login -method=github
+SLACK_WEBHOOK=$(vault read -field=password secret/slack/autoupdate-webhook)
+
+echo '#!/bin/sh
```

Comment:
I feel this would be clearer and easier to edit with a heredoc string (e.g.

```
echo <<< EOF > /usr/local/sbin/check_reboot_required.sh
(all the contents here)
EOF
```

(same with the other big one above)

Reply:
Not sure how you get this to work - my bash script is trying to execute (all the contents here)

Reply:
Ah - << instead of <<< ?

Reply:
No... that creates an empty file...

Reply:
Closest I can get is 
```
tee /etc/apt/apt.conf.d/10periodic <<EOF > /dev/null
(all the contents here)
EOF
```
And I can then put the quotes in literally, but have to back-slash the $

Reply:
(back-slash $ when you want it to stay a $ I mean. The final $SLACK_WEBHOOK isn't back-slashed, because we want to echo the value into the script)

Reply:
sorry wes, the heredoc should be

```
cat << EOF > target
text here
EOF
```